server/server.js

```diff
@@ -191,7 +191,7 @@ app.get('/getingredients', (req, res) => {
 //add banned ingredients for user
 
 app.get('/bannedingredients', (req, res) => {
-  pool.query(`INSERT INTO Banned_Ingredients (user_id, name) VALUES ('${req.query.user_id}', ${req.query.name}');`, (err, rows, fields) => {
+  pool.query(`INSERT INTO Banned_Ingredients (user_id, name) VALUES (${req.query.user_id}, ${req.query.name});`, (err, rows, fields) => {
```

Comment:
You need quotes around the values you're adding to the table because the column types for those values are varchar and string.